conf.py

```diff
@@ -16,6 +16,16 @@
 
 from . import world
 
+class ConfigValidationError(Exception):
+	"""Error when config conditions aren't met."""
+	
```

Comment:
Just this should be adequate (the base Exception class already handles arguments):

```python
class ConfigValidationError(Exception):
    """Error when config conditions aren't met."""
````

Also, please use 4 spaces to indent and not a tab! You can normally tell your editor to prefer this behaviour, at least for Python.